src/serde_helpers.rs

```diff
@@ -4,21 +4,22 @@ use std::{convert::TryFrom, result::Result};
 
 use serde::{ser, Serializer};
 
-pub use bson_datetime_as_iso_string::{
-    deserialize as deserialize_bson_datetime_from_iso_string,
-    serialize as serialize_bson_datetime_as_iso_string,
+pub use bson_datetime_as_rfc3339_string::{
```

Comment:
see RUST-825 for details on these changes. The name was updated to be more specific and accurate per the underlying methods we'd be using, which are RFC 3339 compliant rather than completely ISO 8601. See the docstring for [`parse_from_rfc3339`](https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#method.parse_from_rfc3339) and [this SO post](https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats) for an overview of the subtleties here.